CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v2.2.0](https://github.com/puppetlabs/puppet_operational_dashboards/tree/v2.2.0) (2023-11-16)
+
+[Full Changelog](https://github.com/puppetlabs/puppet_operational_dashboards/compare/v2.1.0...v2.2.0)
+
+### Added
+
+- telegraf: Update Agent 1.27.0-\>1.28.2 [\#197](https://github.com/puppetlabs/puppet_operational_dashboards/pull/197) ([bastelfreak](https://github.com/bastelfreak))
+- Update grafana 8.5.26-\>8.5.27 [\#196](https://github.com/puppetlabs/puppet_operational_dashboards/pull/196) ([bastelfreak](https://github.com/bastelfreak))
+
+### Fixed
+
+- Bugfix for importing new file sync storage metrics [\#200](https://github.com/puppetlabs/puppet_operational_dashboards/pull/200) ([m0dular](https://github.com/m0dular))
+- \(SUP-4138\) Restore missing PCP Connections panel [\#199](https://github.com/puppetlabs/puppet_operational_dashboards/pull/199) ([m0dular](https://github.com/m0dular))
+- \(SUP-4225\) Deprecate Grafana archive installations [\#195](https://github.com/puppetlabs/puppet_operational_dashboards/pull/195) ([m0dular](https://github.com/m0dular))
+- \(SUP-4478\) Addition of access details for Grafana Dashboard [\#193](https://github.com/puppetlabs/puppet_operational_dashboards/pull/193) ([Aaronoftheages](https://github.com/Aaronoftheages))
+
+## [v2.1.0](https://github.com/puppetlabs/puppet_operational_dashboards/tree/v2.1.0) (2023-09-25)
```

Comment:
that's odd. Why wasn't there a changelog entry for the 2.1.0 release previously

Reply:
It's because of https://github.com/github-changelog-generator/github-changelog-generator/issues/971, but I can't figure out why.